doc/source/whatsnew/v2.0.0.rst

```diff
@@ -1306,6 +1306,7 @@ Reshaping
 - Clarified error message in :func:`merge` when passing invalid ``validate`` option (:issue:`49417`)
 - Bug in :meth:`DataFrame.explode` raising ``ValueError`` on multiple columns with ``NaN`` values or empty lists (:issue:`46084`)
 - Bug in :meth:`DataFrame.transpose` with ``IntervalDtype`` column with ``timedelta64[ns]`` endpoints (:issue:`44917`)
+- Bug in :func:`agg` would ignore arguments when passed a list of functions (:issue:`50863`)
```

Comment:
The impacted methods are ``:meth:`DataFrame.agg` ``, ``:meth:`Series.agg` ``, ``:meth:`DataFrameGroupBy.agg` ``,  ``:meth:`SeriesGroupBy.agg` ``, and ``:meth:`Resampler.agg` ``. Could you add tests for each of these (so far only testing DataFrame).

Reply:
Sure. Does the content in whatsnew need to be changed, or do we just need to add more tests?

Reply:
I think the line in the whatsnew should reference the methods listed.

Reply:
cc @rhshadrach

I added these tests and changed whatsnew.

But unfortunately the same problem exists in `GroupyBy`, I tried to make a small correction, they look like they work fine now, any suggestions?